src/config/services/user-services.md

```diff
@@ -13,7 +13,13 @@ following `run` script:
 ```
 #!/bin/sh
 
-exec chpst -u "<username>:$(id -Gn <username> | tr ' ' ':')" runsvdir /home/<username>/service
+export USER="${PWD##*-}"
```

Comment:
I would prefer to not derive the user from PWD as this adds more moving parts to it, users might mess up.

Reply:
I think picking a username from PWD is OK as long as it doesn't replace the login-exported USER and isn't exported to the environment. This lets one generic `run` script be used for multiple users, the way the `agetty` services all link back to `agetty-generic/run`.

Thus, I would call the variable `user` or `_user` and do not export.

Reply:
The whole point of the change is to export USER.

I think a generic runsvdir script is ok, but this requires more documentation for the details and I don't think its really worth it.

Reply:
```suggestion
export USER="<username>"
```

Reply:
Here's another, more robust, alternative:
```suggestion
export USER="${PWD##*/runsvdir-}"
```
Because of the `/` this works in any case. Keep the `##` so it works even if the path contains a directory named `runsvdir-*`.

Would that be ok? I can amend the commit with the version you prefer.

Reply:
I personally would prefer hardcoded username because explaining generic services feels out of scope and unnecessary for most users.

Reply:
I agree, done that.